lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -379,12 +423,12 @@ func (plmgr *PluginManager) handleMisbehavingPlugins(
 
         // check misbehaving plugin
         if pluginmetadata.CheckMisbehavingPlugins(pluginKey) {
-            lomcommon.LogInfo("In handleRequest(): Plugin %v is misbehaving for anamoly key %v. Ignoring the response",
+            LogInfo("In handleMisbehavingPlugins(): Plugin %v is misbehaving for anamoly key %v. Ignoring the response",
                 respData.Action, respData.AnomalyKey)
 
             pluginmetadata.SetPluginStage(plugins_common.PluginStageDisabled)
 
-            lomcommon.GetGoroutineTracker().Start("handleMisbehavingPlugins"+respData.Action+"_"+lomcommon.GetUUID(),
+            lomcommon.GetGoroutineTracker().Start("handleMisbehavingPlugins_"+respData.Action+"_"+lomcommon.GetUUID(),
                 func() {
                     // stop the plugin
                     plmgr.shutdownPlugin(respData.Action)
```

Comment:
SHould not need goroutine for shutdown plugin. Refer its comments

Reply:
remove it

Reply:
TODO